spring-cloud-alibaba-examples/appactive-example/common/src/main/java/com/alibaba/cloud/example/common/RPCType.java

```diff
@@ -25,6 +25,11 @@ public enum RPCType {
 	/**
 	 * Spring Cloud.
 	 */
-	SpringCloud
+	SpringCloud,
+
+	/**
+	 * Dubbo.
+	 */
+	Dubbo
```

Comment:
Please delete `Dubbo`. We shouldn't include it.

Reply:
> 请删除`Dubbo`。我们不应该包括它。

If you delete it from the project, the project will not run properly!